python/tk_desktop/notifications/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from .startup_update_notification import StartupUpdateNotification # noqa
```

Comment:
`# noqa` tells pycodestyle that errors on this line should be ignore. In this case here, we have an unused import.